examples/docs/en-US/upload.md

```diff
@@ -116,6 +116,15 @@
       },
       handleExceed(files, fileList) {
         this.$message.warning(`You can upload up to 3 files. You selected ${files.length} files this time, and ${files.length + fileList.length} files totally`);
+      },
+      handleBeforeRemove(file, fileList) {
+        return new Promise((resolve, reject) => {
+          if (confirm('It looks good, are you sure to remove?')) {
+            resolve()
```

Comment:
Please add a semicolon here. Also applies to the other 7 `resolve()`s and `reject()`s.